pkg/ovsdb/transact.go

```diff
@@ -1020,8 +1023,8 @@ func doSelect(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 		if err != nil {
 			return err
 		}
-		ovsResult.Count = ovsResult.Count + 1
-		ovsResult.Rows = append(ovsResult.Rows, *resultRow)
+		*ovsResult.Count++
```

Comment:
before going into the for loop, we have to create the rows set.
*ovsResult.Rows = []libovsdb.Row{}
you don't need Count here

Reply:
ok